cv_next/app/layout.tsx

```diff
@@ -32,7 +32,12 @@ export default function RootLayout({ children }: RootLayoutProps) {
         <ThemeProvider attribute="class" defaultTheme="system" enableSystem>
           <CvsProvider>
             <Navbar />
-            <div className="container mx-auto space-y-8 p-6">{children}</div>
+            <div
+              className="container mx-auto max-w-full space-y-8 p-6"
+              style={{ width: "85%" }}
```

Comment:
Is that good for mobile as well?

Reply:
Made it so that the phone will look the same and bigger screens will get the new settings